src/dsymbol/conversion/first.d

```diff
@@ -64,15 +65,15 @@ final class FirstPass : ASTVisitor
 	 */
 	this(const Module mod, istring symbolFile, IAllocator symbolAllocator,
 		IAllocator semanticAllocator, bool includeParameterSymbols,
-		ModuleCache* cache, CacheEntry* entry = null)
+		ModuleCache* cache, CacheEntry* entry = null) nothrow
```

Comment:
This change seems to be unrelated to this PR.

Reply:
That avoided a warning.